build_protos.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2017 Google Inc.
```

Comment:
Can we put this somewhere other than in the root directory? Would a directory in scripts/build/ or build/ work?